pkg/Library/Inspector/Atoms/NodeInspectorAdaptor.js

```diff
@@ -11,10 +11,11 @@ async update({graph, selected}, state, tools) {
   if (!deepEqual(graph, state.graph) || (selected !== state.selected)) {
     state.selected = selected;
     state.graph = graph;
-    return this.updateData({graph, selected}, state, tools);
+    return this.updateData(state, tools);
```

Comment:
this was the weirdest experience ever, that i could only reproduce in incognito tab:
if `selected` was `undefined`, it would be passed to the `updateData` method as `"undefined"` - a string... 🤷

Reply:
i cannot reproduce it anymore... it makes me happy on one hand (sanity is restored), makes me sad otoh (that's the part where i start doubting my own sanity... i should have made a video of this!)

anyway... i can revert this file (also can keep it, because there is nothing wrong or weird with the code as it is now). lmk what you think is best.